packages/ui/src/views/repo/repo-branch/components/divergence-gauge.tsx

```diff
@@ -25,54 +27,52 @@ export const DivergenceGauge = ({ behindAhead, className }: GaugeProps) => {
+        <Text variant="body-single-line-normal" align="right">
           {behindAhead.behind ?? 0}
-          <span className="sr-only">
-            {t('views:repos.commits', 'commits')}
-            {t('views:repos.behind', 'behind')}
-          </span>
-        </span>
+          <span className="sr-only">{t('views:repos.branches.commitsBehind', 'commits behind')}</span>
+        </Text>
         <Separator orientation="vertical" />
-        <span className="truncate text-2 leading-none text-cn-foreground-3">
+        <Text variant="body-single-line-normal">
           {behindAhead.ahead ?? 0}
-          <span className="sr-only">
-            {t('views:repos.commits', 'commits')}
-            {t('views:repos.ahead', 'ahead')}
-          </span>
-        </span>
-      </div>
+          <span className="sr-only">{t('views:repos.branches.commitsAhead', 'commits ahead')}</span>
+        </Text>
+      </Layout.Grid>
       {/* Both behind and ahead are 0, don't show the progress bar */}
       {/* TODO: replace with meter component when available */}
       {behindAhead?.behind === 0 && behindAhead?.ahead == 0 ? null : (
-        <div className="mx-auto flex w-28 items-center justify-center">
-          <div className="flex w-1/2 flex-row-reverse justify-start">
-            <div style={{ width: `${adjustedBehindPercentage}%` }}>
-              <Progress
-                className={`rotate-180 [&_.cn-progress-root::-moz-progress-bar]:bg-cn-background-8 [&_.cn-progress-root::-webkit-progress-value]:bg-cn-background-8 [&_.cn-progress-root]:rounded-l-none`}
-                value={adjustedBehindPercentage / 100}
-                size="sm"
-                hideIcon
-                hidePercentage
-                hideContainer
-              />
-            </div>
-          </div>
-          <div className="flex w-1/2 justify-start">
-            <div style={{ width: `${adjustedAheadPercentage}%` }}>
-              <Progress
-                className={`[&_.cn-progress-root::-moz-progress-bar]:bg-cn-background-13 [&_.cn-progress-root::-webkit-progress-value]:bg-cn-background-13 [&_.cn-progress-root]:rounded-l-none`}
-                value={adjustedAheadPercentage / 100}
-                size="sm"
-                hideIcon
-                hidePercentage
-                hideContainer
-              />
-            </div>
-          </div>
-        </div>
+        <Layout.Grid columns={2} align="center" justify="center" className="mx-auto w-20">
+          <Progress
+            className="rotate-180 justify-self-end [&_.cn-progress-root]:rounded-l-none"
+            value={adjustedBehindPercentage / 100}
+            size="sm"
+            hideIcon
+            hidePercentage
+            hideContainer
+            style={
+              {
+                width: `${adjustedBehindPercentage}%`,
+                '--cn-progress-bar-bg-color': 'var(--cn-set-gray-soft-bg)'
```

Comment:
can we avoid using css variables in react component? instead can we add different class for `var(--cn-set-gray-soft-bg)`?

Reply:
No worries, I rolled back my solution ⏮️